Documentation/templates/MigrationGuide.md

```diff
@@ -6,6 +6,7 @@
 * XCAssets #480: removed `noAllValues` parameter, replaced by `allValues` parameter.
 * XCAssets #482: removed deprecated `allValues` constant
 * XCAssets #482: changed x-platform image alias default from `Image` to `AssetImageTypeAlias`.
+* IB #419: Split up the templates into one for scenes and one for segues (in preparation for future templates for accessibility labels, ...).
```

Comment:
I guess that for now that part in the `MigrationGuide.md` is just a scratchpad, but in the final version we should clearly document the impact, especially for people using segues how they would need to modify their config files to have 2 { template, output } pairs instead of one (especially since in that breaking change version we also changed the config file format anyway)

Reply:
Yeah, exactly. I'm trying to just collect all the necessary info to write nice docs later (this WE?)